src/create-mollie-api.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { AxiosInstance } from 'axios';
+
+import PaymentsResource from './resources/payments';
+import PaymentsRefundsResource from './resources/payments/refunds';
+import PaymentsChargebacksResource from './resources/payments/chargebacks';
+import MethodsResource from './resources/methods';
+import RefundsResource from './resources/refunds';
+import CustomersResource from './resources/customers';
+import CustomersPaymentsResource from './resources/customers/payments';
+import CustomersMandatesResource from './resources/customers/mandates';
+import CustomersSubscriptionsResource from './resources/customers/subscriptions';
+import ChargebacksResource from './resources/chargebacks';
+import OrdersShipmentsResource from './resources/orders/shipments';
+import OrdersRefundsResource from './resources/orders/refunds';
+import OrdersResource from './resources/orders';
+import OrdersLinesResource from './resources/orders/lines';
+import PaymentsCapturesResource from './resources/payments/captures';
+
+interface ICreateMollieApi {
+  httpClient: AxiosInstance;
+}
+
+export interface IMollieApiClient {
+  payments: PaymentsResource;
+  methods: MethodsResource;
+  payments_refunds: PaymentsRefundsResource;
+  refunds: RefundsResource;
+  payments_chargebacks: PaymentsChargebacksResource;
+  chargebacks: ChargebacksResource;
+  payments_captures: PaymentsCapturesResource;
+  customers: CustomersResource;
+  customers_payments: CustomersPaymentsResource;
```

Comment:
```suggestion
  customer_payments: CustomerPaymentsResource;
```

Reply:
Same for `PaymentsCapturesResource` above, should be singular `Payment` 🙂